src/kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  FRONTEND_BASE_URL : "https://moneyclaims.aat.platform.hmcts.net"
```

Comment:
These values need to come from citizen frontend config in aat not from the claim-store value,

you can retrieve them from the app with:
```
az account set -s 1c4f0704-a29e-403d-b719-b90c34ef14c9
az webapp config appsettings list  --resource-group cmc-citizen-frontend-aat --name cmc-citizen-frontend-aat  --query '[].{ Name: name, Value: value }' -o tsv
```

make sure you don't put anything that looks like a password in here it should go in the deployment.template as a variable and it'll be retrieved from vault if you add the config in the Jenkinsfile_CNP